src/components/EditableTypography/EditableTypography.tsx

```diff
@@ -40,18 +44,19 @@ const EditableTypography: VibeComponent<EditableTypographyProps, HTMLElement> =
       ariaLabel = "",
       placeholder,
       typographyClassName,
-      renderTypography
+      component: TypographyComponent
     },
     ref
   ) => {
     const componentRef = useRef(null);
     const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
 
-    const [isEditing, setIsEditing] = useState(false);
+    const [isEditing, setIsEditing] = useState(null);
```

Comment:
why is `false` doesn't get the job done?

Reply:
Good catch, useless now, was used in previous attempts, thanks

Reply:
Make sure that it doesn't break anything, maybe it was intended
Notice the `useEffect` where you check specifically for nullish, not false